### PR TITLE
Fix some pylint errors

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
pylint catches a lot of errors if it's ok to fix all errors I can send more patches.

### CODE REVIEW

**Comment by dralley** [comment]
Please add ```[noissue]``` to the bottom of your commit message

Otherwise, looks good to me

**Comment by alexandrevicenzi** [comment]
@dralley done.

File: pulpcore/pulpcore/app/serializers/__init__.py [hunk 1]
```diff
@@ -7,7 +7,6 @@
     ModelSerializer,
     MasterModelSerializer,
     DetailIdentityField,
-    DetailRelatedField,
```
Comment (dralley) [inline pulpcore/pulpcore/app/serializers/__init__.py]: One more thing, remove this one change please

Comment (alexandrevicenzi) [reply pulpcore/pulpcore/app/serializers/__init__.py]: Why? This import was duplicated. I didn't remove the import, it was just duplicated.

Comment (dralley) [reply pulpcore/pulpcore/app/serializers/__init__.py]: Ah, you're right.  For some reason (probably severe exhaustion) I was thinking it was an "unused import" or something.  Even though I saw that it was duplicated the day before.

Anyway... this looks good to merge.  Thanks @alexandrevicenzi

**Pull request merged**